protocol/x/clob/types/untriggered_conditional_orders.go

```diff
@@ -0,0 +1,83 @@
+package types
+
+import "bytes"
+
+// MinConditionalOrderHeap is type alias for `[]Order` which implements "container/heap"
+// interface.
+//
+// This is a _MIN_ heap. Orders are compared by their `ConditionalOrderTriggerSubticks` field, and then by
+// their `OrderHash` if the trigger subticks are equal.
+type MinConditionalOrderHeap []Order
+
+func (h MinConditionalOrderHeap) Len() int {
+	return len(h)
+}
+
+func (h MinConditionalOrderHeap) Less(i, j int) bool {
+	x, y := h[i], h[j]
+
+	// If the trigger subticks are the same, sort by order hash.
+	// This is required for determinism in the case of multiple orders with the same trigger subticks.
```

Comment:
I think technically we don't need this since orders will get added deterministically in the same order, but I added this to be 100% safe. Let me know if anyone disagrees